plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/CloudSdkStagingHelperTest.java

```diff
@@ -69,20 +69,22 @@
 
   @Mock private IProgressMonitor monitor;
 
+  private IPath sourceDirectory;
   private IPath stagingDirectory;
   private IProject project;
 
   @Before
   public void setUp() {
     project = projectCreator.getProject();
     stagingDirectory = new Path(tempFolder.getRoot().toString());
+    sourceDirectory = new Path(tempFolder.getRoot().toString());
   }
 
   @Test
   public void testStage_cancelled() throws AppEngineException {
     when(monitor.isCanceled()).thenReturn(true);
     try {
-      CloudSdkStagingHelper.stageStandard(null, null, null, monitor);
+      CloudSdkStagingHelper.stageStandard(null, stagingDirectory, null, monitor);
```

Comment:
Was there an issue previously?

Reply:
yes, this argument isn't nullable if I recall